creator-node/scripts/updateContentBlacklist.js

```diff
@@ -50,40 +50,43 @@ async function run () {
     return
   }
 
-  console.log('Updating Content Blacklist...\n')
+  console.log(`Updating Content Blacklist for ${CREATOR_NODE_ENDPOINT}...\n`)
   const { action, type, values, verbose } = args
-  try {
-    switch (action) {
-      case 'ADD': {
-        await addToContentBlacklist(type, values)
-        break
-      }
-      case 'REMOVE': {
-        await removeFromContentBlacklist(type, values)
-        break
-      }
-      default: {
-        console.error('Should not have reached here :(')
-        return
+  for (let i = 0; i < values.length; i += 10) {
```

Comment:
can we make the pagination slice a const at the top of the file, and update line 56 with it too

Reply:
was there just too many things to process and a pagination was necessary?